File: gb-angular-frontend/src/app/interfaces/withdrawal-admin.ts
export interface WithdrawalAdmin {
  withdrawal_id: number;
  user_id: number;
  username: string;
  email: string;
  holder_name: string;
  account_number: string;
  bank_name: string;
  branch_name: string;
  approval_date: string;
  request_date: string;
  amount: number;
  payment_date: string;
  status: string;
}

File: gb-spring-backend/src/main/java/com/egroupbuy/dto/WithdrawalAdmin.java
package com.egroupbuy.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class WithdrawalAdmin {
    private int withdrawal_id;
    private int user_id;
    private String username;
    private String email;
    private String holder_name;
    private String account_number;
    private String bank_name;
    private String branch_name;
    private Timestamp request_date;
    private double amount;
    private Timestamp approval_date;
    private Timestamp payment_date;
    private String status;
}
